virttest/test_setup.py

```diff
@@ -2034,9 +2037,12 @@ def cleanup(self):
             if os.path.exists(self.libvirtd_backup_path):
                 os.rename(self.libvirtd_backup_path, self.libvirtd_path)
             if self.user.count('EXAMPLE'):
-                logging.debug("Delete the created user 'testacl'.")
-                cmd = "userdel -r testacl"
+                self.user = 'testacl'
```

Comment:
Then in that case we should not ideally do this, because if we run in workstation testacl should not be hardcoded user. always it should be taking from param `unprivileged_user`

Reply:
sorry, not sure you mean, we need set a unprivileged_user in case cfg file?

Reply:
Right, I will try to rephrase my opinion,
self.user is initialized in `__init__` with param `unprivileged_user` that we set in case cfg file
so during cleanup why do we hardcode self.user ?

Reply:
Not sure why due to from this class doc
"""
Create a non-privileged user 'testacl' for test if given
    'unprivileged_user' contains 'EXAMPLE', and delete the user at cleanup.
"""
I just want to create a user due to the old code will not create the user if self.user is not EXAMPLE, and dont delete the user if the use is not created by script.